### DISCUSSION TITLE
Can I specify team ID for the Slack provider?

### DISCUSSION THREAD

**Discussion by zoul** [nextauthjs/next-auth#7566] [Help] [answered]
### Question 💬

When people are signed into multiple Slack worspaces, the Slack OAuth flow can get confused about what team you are trying to sign into using NextAuth. Slack says it’s possible to include a `team` argument when posting to `https://slack.com/oauth/authorize` to explicitly specify the Slack team you want to get signed into. Can I make NextAuth to include this argument? Looking at the docs and the types I can’t find it anywhere.

### How to reproduce ☕️

I don’t have a clear repro for this, the user has to be signed into multiple Slack workspaces for the problem to arise.

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by balazsorban44** [comment, marked as answer]
If you mean https://api.slack.com/legacy/oauth#send_users

than you can pass `team` to the `signIn` method and we will forward it https://next-auth.js.org/getting-started/client#additional-parameters

**Reply by zoul** [reply]
Thank you very much, that’s it, I have missed it in the docs.

**Reply by Starefossen** [reply]
@balazsorban44 this does not seam to work as expected, is this still supported?
